packages/web/babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Next.js added support for Relay in their compiler

https://nextjs.org/docs/advanced-features/compiler#relay